vllm/model_executor/models/minicpmv.py

```diff
@@ -57,6 +70,957 @@
     "language_model.model": "language_model",
 }
 
+logger = logging.getLogger("vllm")
+
+
+# For Siglip: copied from  HuggingFaceM4/siglip-so400m-14-980-flash-attn2-navit and add tgt_sizes
+class SiglipVisionConfig(PretrainedConfig):
```

Comment:
Let's move `SiglipVisionModel` and its components to a separate file since we're planning to add it in anyway according to the RFC (#4194).